### ISSUE TITLE
Error: /bin/sh: 1: az: not found in v1.0.5

### ISSUE THREAD

**Issue by jnsvd** [Azure/cli#62]
We're getting the following output in the pipeline run logs since `v1.0.5` is released:

```
Run azure/CLI@v1
  with:
    inlineScript: az login --identity --only-show-errors
  echo "TOKEN=$(az acr login --name <redacted>.azurecr.io \
      --expose-token \
      --output tsv \
      --query accessToken \
      --only-show-errors)" >> $GITHUB_ENV

    azcliversion: agentazcliversion
  env:
    EXPOSE_DOMAIN_NAME: <redacted>
    ACR_RELEASE: <redacted>.azurecr.io
    ENVIRONMENT_REPOSITORY_STAGING: <redacted>
    VERSION: 2.0.259
    POI_REGISTRY: <redacted>.azurecr.io
    JAVA_HOME: /opt/hostedtoolcache/Java_Adopt_jdk/11.0.13-8/x64
Error: Error: Command failed: az version
/bin/sh: 1: az: not found

Error: /bin/sh: 1: az: not found

Warning: Error: EISDIR: illegal operation on a directory, unlink '/runner/_work/_temp'
cleaning up container...
Warning: Error: No such container: MICROSOFT_AZURE_CLI_1636963334349_CONTAINER

(node:12389) UnhandledPromiseRejectionWarning: Error: Command failed: az version
/bin/sh: 1: az: not found

    at ChildProcess.exithandler (child_process.js:295:12)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:430:11)
    at Socket.emit (events.js:210:5)
    at Pipe.<anonymous> (net.js:659:12)
(node:12389) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:12389) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

When using previous version `1.0.4` (by configuring `uses: azure/CLI@1.0.4`) no such errors are observed and the build is completing successfully.

**Comment by t-dedah** [comment]
Hi @jnsvd We released a new version v1.0.6 with a fix for the above issue. Please test and let us know if you are still facing the issue. 

PS - v1 is now pointing to v1.0.6 so the above workflow requires no changes.

**Comment by jnsvd** [comment]
@t-dedah I confirm the fix is working, thank you for the fast response!
